samples/BasicXrApp/CubeGraphics.cpp

```diff
@@ -379,8 +379,8 @@ namespace {
             ID3D11RenderTargetView* renderTargets[] = {renderTargetView.Get()};
             m_deviceContext->OMSetRenderTargets((UINT)std::size(renderTargets), renderTargets, depthStencilView.Get());
 
-            const XMMATRIX spaceToView = XMMatrixInverse(nullptr, LoadXrPose(layerView.pose));
-            XMMATRIX projectionMatrix = ComposeProjectionMatrix(layerView.fov, {0.05f, 100.0f});
+            const DirectX::XMMATRIX spaceToView = XMMatrixInverse(nullptr, LoadXrPose(layerView.pose));
+            DirectX::XMMATRIX projectionMatrix = ComposeProjectionMatrix(layerView.fov, {0.05f, 100.0f});
```

Comment:
Why are some things prefixed with DirectX:: but not other things (like XMStoreFloat4x4)?

Reply:
These prefixes are added due to a leak of "using namespace" in original math header.   I will double check if there's another leak of this global namespace in header.

Reply:
It seems these functions are leaked to global namespace through "directxmathconvert.inl" inside "directxmath.h" itself.  I will try to keep this consistent, but it is a bug (or feature) from dxmath itself.